GitUI/CommandsDialogs/FormResolveConflicts.Designer.cs

```diff
@@ -329,9 +330,9 @@ private void InitializeComponent()
             label7.Location = new Point(3, 0);
             label7.Name = "label7";
             label7.Padding = new Padding(0, 3, 0, 3);
-            label7.Size = new Size(37, 22);
+            label7.Size = new Size(80, 21);
             label7.TabIndex = 1;
-            label7.Text = "Local";
+            label7.Text = "Local/Current";
```

Comment:
In dependency on the operation:

```suggestion
            label7.Text = "Local/current (ours)";
```

```suggestion
            label7.Text = "Local/current (theirs)";
```

Reply:
Done. Required a refactoring (introducing class `DisplayWithSuffixUpdater`) to do the same thing we did to display keyboard shortcuts in tooltips.

Also:
* added a tooltip:
![image](https://github.com/gitextensions/gitextensions/assets/460196/c3092b98-323a-4201-ae8a-90ab1897830a)

& 

![image](https://github.com/gitextensions/gitextensions/assets/460196/a6d71cf9-92ff-4a2f-a37c-5109fd196624)
* fixed documentation link that was broken
* 2 other opportune refactorings